platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -570,4 +570,14 @@ protected BundleContext getContext() {
     void setServiceTracker(ServiceTracker serviceTracker) {
         this.serviceTracker = serviceTracker;
     }
+
+    /**
+     * mBeanServer setter method. Needed for use in unit tests.
+     *
+     * @param mBean
+     *             the desired mBean instance
+     */
+    void setMBeanServer(MBeanServer mBean) {
```

Comment:
Instead of adding a setter for this, you could also add a new parameter to the constructor and update the blueprint.xml file like this:
```
    <bean id="mbeanServer"
        class="java.lang.management.ManagementFactory"
        factory-method="getPlatformMBeanServer"/>

    <bean id="appServiceMBean"
          class="org.codice.ddf.admin.application.service.impl.ApplicationServiceBean"
          init-method="init"
          destroy-method="destroy">
        <argument ref="appService"/>
        <argument ref="configurationAdminExt"/>
        <argument ref="mbeanServer"/>
        <property name="applicationPlugins" ref="applicationPluginList"/>
    </bean>
```
You can then pass the mock to constructor in your unit test.